frontend/src/bundles/studio/store/slice.ts

```diff
@@ -121,6 +129,12 @@ const { reducer, actions, name } = createSlice({
                 items: state.scripts,
             });
         },
+        changeScriptVoice(state, action: PayloadAction<ScriptVoicePayload>) {
+            const { scriptId, voice } = action.payload;
+            state.scripts = state.scripts.map((script) =>
+                script.id === scriptId ? { ...script, voice } : script,
+            );
+        },
```

Comment:
Please, reuse editScript reducer for changing voice. And we are going to use `voiceName` property https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/236